frontend/src/pages/Sessions/SessionsFeedV3/MinimalSessionCard/utils/utils.ts

```diff
@@ -2,8 +2,13 @@ import { Maybe, Session } from '@graph/schemas'
 import { H } from 'highlight.run'
 import validator from 'validator'
 
+type SessionWithIdentityInformation = Pick<
+	Session,
+	'user_properties' | 'identifier' | 'fingerprint'
+>
+
 export const getIdentifiedUserProfileImage = (
-	session?: Maybe<Partial<Session>>,
```

Comment:
Got a weird typescript error after the following field was added to the Session type in the frontend ```generated/schemas.tsx```

```typescript
session_comments?: Maybe<Array<Maybe<SessionComment>>>
```

This was the "solution/workaround" that satisfied the ts compiler.